docs/background.md

```diff
@@ -0,0 +1,368 @@
+
+$$
+\begin{align*} % Use the align environment for several lines separated by double backslash
+\mu_0 (x) &:= \mathbb{E}[Y(0) | X = x] \\
+\mu_1 (x) &:= \mathbb{E}[Y(1) | X = x]
+\end{align*}
+$$
+
+$\hat{\mu}_0$ and $\hat{\mu}_1$ are estimated using the untreated and treated observations respectively.
+
+Then the CATE is estimated as:
+
+$$
+\hat{\tau}^T(x) := \hat{\mu}_1(x) - \hat{\mu}_0(x)
+$$
+
+#### More than binary treatment
+
+In the case of multiple discrete treatments, one estimator is trained for each treatment variant (including the control):
+
+$$
+\mu_k (x) := \mathbb{E}[Y(k) | X = x] \; \forall k \in \{0,\dots, K-1\}
+$$
+
+The CATE for each treatment variant against the control is then estimated with:
+
+$$
+\hat{\tau}_k^T(x) := \hat{\mu}_k(x) - \hat{\mu}_0(x) \; \forall k \in \{1,\dots, K-1\}
+$$
+
+### X-Learner
+
+The X-Learner was introduced by [Kuenzel et al. (2019)](https://arxiv.org/pdf/1706.03461.pdf). It is an extension of the T-Learner and consists of three stages:
+
+1. Estimate the conditional average outcomes for each variant:
+
+    $$
+    \begin{align*}
+    \mu_0 (x) &:= \mathbb{E}[Y(0) | X = x] \\
+    \mu_1 (x) &:= \mathbb{E}[Y(1) | X = x]
+    \end{align*}
+    $$
+
+1. Impute the treatment effect for the observations in the treated group based on the control-outcome estimator as well as the treatment effect for the observations in the control group based on the treatment-outcome estimator:
+
+    $$
+    \begin{align*}
+    \widetilde{D}\_1^i &:= Y^i_1 - \hat{\mu}\_0(X^i_1) \\
+    \widetilde{D}\_0^i &:= \hat{\mu}\_1(X^i_0) - Y^i_0
+    \end{align*}
+    $$
+
+    Then estimate \(\tau_1(x) := \mathbb{E}[\widetilde{D}_1^i | X=x]\) and \(\tau_0(x) := \mathbb{E}[\widetilde{D}_0^i | X=x]\) using the observations in the treatment group and the ones in the control group respectively.
+
+1. Define the CATE estimate by a weighted average of the two estimates in stage 2:
+
+    $$
+    \hat{\tau}^X(x) := g(x)\hat{\tau}_0(x) + (1-g(x))\hat{\tau}_1(x)
+    $$
+
+    Where \(g(x) \in [0,1]\). We take \(g(x) := \mathbb{E}[W = 1 | X=x]\) to be the propensity score.
+
+#### More than binary treatment
+
+In the case of multiple discrete treatments, the stages are similar to the binary case:
+
+1. One outcome model is estimated for each variant (including the control), and one propensity model is trained as a multiclass classifier, $\forall k \in \{0,\dots, K-1\}$:
+
+    $$
+    \begin{align*}
+    \mu_k (x) &:= \mathbb{E}[Y(k) | X = x]\\
+    e(x, k) &:= \mathbb{E}[\mathbb{I}\{W = k\} | X=x] = \mathbb{P}[W = k | X=x]
+    \end{align*}
+    $$
+
+1. The treatment effects are imputed using the corresponding outcome estimator, $\forall k \in \{1,\dots, K-1\}$:
+
+    $$
+    \begin{align*}
+    \widetilde{D}\_k^i &:= Y^i \cdot k - \hat{\mu}_0(X^i \cdot k) \\
```

Comment:
```suggestion
    \widetilde{D}_k^i &:= Y^i_k - \hat{\mu}_0(X^i_k) \\
```
See the docs on `main` for reference:
<img width="174" alt="Screenshot 2025-02-22 at 2 21 50 PM" src="https://github.com/user-attachments/assets/e16c15da-49d0-4dba-9c8b-16fe3224c48b" />